README.md

```diff
@@ -138,8 +139,8 @@ Where:
      The number of seconds that the test should run. Default: 5.
 
    --connections <uint32_t>
-     The number of connections per event loop that the test should
-     maximally use. HTTP/1 only. Default: 1.
+     The maximum allowed number of concurrent connections per event loop.
+     HTTP/1 only. Default: 1.
```

Comment:
Thinking about it, I think we should to set the defaults to "what would give the same result that some really simple `while` loop QPS generator would on a redline` test. Many folks will just want to measure max QPS (even though this is not the meaningful IMHO). To make this comparable to other tools, it should work for this out-of-the-box. The other settings can be used for more sensitive experiments, like latency vs. QPS characterizaiton.

Reply:
So one the one hand, if someone downloads the tool and goes `./nighthawk_client arbitrarydomain.com` to try it out, it would be nice if that didn't result in a dead server or blacklisting :-) That is what motivated lowish defaults. To minimize friction for common cases, to float an idea, how about adding flags like `--run-XXX-test` (where XXX could be max-rps, which could then perhaps load a packaged `max-rps-test-options.proto` from disk) to reduce friction?

Reply:
(but, arguably, `--connections` could be defaulted higher with the current create-as-needed- strategy, as the low rps and duration will probably prevent problems there)

Reply:
I bumped the default for `--connections` to `100` as that will allow putting a decent amount of pressure on most servers by just tweaking `--rps`. @htuch @jplevyak does that sound good to you?